react/src/lib/components/DeckGLMap/layers/colormap/colormapLayer.ts

```diff
@@ -103,6 +111,9 @@ export interface ColormapLayerProps extends BitmapLayerProps {
 
     // check for logarithmic values
     isLog?: boolean;
+
+    // // check for nearest interpolation
+    isNearest?: boolean;
```

Comment:
I don't think we should need these settings. The `colorMapFunction` should cover interpolation as well as breakpoints.